From bf949f570a232423c7cf01831dfbe7034a4f49d8 Mon Sep 17 00:00:00 2001
Message-Id: <bf949f570a232423c7cf01831dfbe7034a4f49d8@dist-git>
From: Peter Krempa <pkrempa@redhat.com>
Date: Tue, 31 Jan 2023 15:23:54 +0100
Subject: [PATCH] qemu_fd: Remove declaration for 'qemuFDPassNewDirect'

The function doesn't exist any more.

Signed-off-by: Peter Krempa <pkrempa@redhat.com>
Reviewed-by: Martin Kletzander <mkletzan@redhat.com>
(cherry picked from commit 51dc38fe31beb252cc0fa2780210cdedc698f57f)

https://bugzilla.redhat.com/show_bug.cgi?id=2040272
---
 src/qemu/qemu_fd.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/qemu/qemu_fd.h b/src/qemu/qemu_fd.h
index 6f165b6be9..032b9442ee 100644
--- a/src/qemu/qemu_fd.h
+++ b/src/qemu/qemu_fd.h
@@ -30,9 +30,6 @@ G_DEFINE_AUTOPTR_CLEANUP_FUNC(qemuFDPass, qemuFDPassFree);
 qemuFDPass *
 qemuFDPassNew(const char *prefix,
               void *dompriv);
-qemuFDPass *
-qemuFDPassNewDirect(const char *prefix,
-                    void *dompriv);
 
 void
 qemuFDPassAddFD(qemuFDPass *fdpass,
-- 
2.39.1

